extensions/data-transfer/portability-data-transfer-flickr/src/main/java/org/datatransferproject/datatransfer/flickr/photos/FlickrPhotosExporter.java

```diff
@@ -123,6 +123,14 @@ static String toMimeType(String flickrFormat) {
 
     RequestContext.getRequestContext().setAuth(auth);
 
+    if (exportInformation.isPresent()
```

Comment:
Let's add a comment here to describe this logic

Reply:
I added a comment, but it was below the if statmenet, moved it to the top